src/wallet3/cli-wallet/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = 'oxen_wallet_cli'
+authors = [
+    {name = "Your Name", email = "you@yourdomain.com"},
```

Comment:
Should fill this out, either yourself, or the generic `"Oxen Project", `"team@oxen.io"`.